web/src/views/DandisetLandingView/DandisetOwnersDialog.vue

```diff
@@ -94,7 +91,16 @@ const userFormatConversion = (users) => users.map(({
   id: _id, level: _accessLevel, login, name: `${firstName} ${lastName}`,
 }));
 
-const addResult = (users) => users.map((u) => ({ ...u, result: `${u.name} (${u.login})` }));
+// eslint-disable-next-line camelcase
+const girderize = (users) => users.map(({ username, first_name, last_name }) => ({
+  id: username,
+  login: username,
+  username,
+  // eslint-disable-next-line camelcase
+  name: (first_name && last_name) ? `${first_name} ${last_name}` : null,
+}));
+
+const addResult = (users) => users.map((u) => ({ ...u, result: (u.name) ? `${u.name} (${u.login})` : u.login }));
```

Comment:
A comment explaining why this is neccesary might be helpful. You might also consider renaming this to `injectResult`, since `addResult` makes it seem like you're adding an element to the list, but it's a very minor thing.